### DISCUSSION TITLE
/etc/sysctl.d/99-reserved-ports.conf has incorrect syntax

### DISCUSSION THREAD

**Discussion by trocade** [Security-Onion-Solutions/securityonion#3276] [Unsupported Versions] [answered]
I noticed that all my SO2 nodes had a failure on systemd-sysctl and the issue is /etc/sysctl.d/99-reserved-ports.conf

The config that is set during installation are:
`net.ipv4.ip_local_reserved_ports="55000,57314"`

This syntax works directly with sysctl -w but not running sysctl -p
```
cat /etc/sysctl.d/99-reserved-ports.conf
net.ipv4.ip_local_reserved_ports="55000,57314"
sysctl -p /etc/sysctl.d/99-reserved-ports.conf
sysctl: setting key "net.ipv4.ip_local_reserved_ports": Invalid argument
net.ipv4.ip_local_reserved_ports = "55000,57314"
```

Removing the `""` corrects this:

```
cat /etc/sysctl.d/99-reserved-ports.conf
net.ipv4.ip_local_reserved_ports=55000,57314
sysctl -p /etc/sysctl.d/99-reserved-ports.conf
net.ipv4.ip_local_reserved_ports = 55000,57314
```
I tested this on both Ubuntu 18 and Centos 7.9 and got the same behaviour.

**Comment by jertel** [comment, marked as answer]
Thanks for reporting this. I've opened https://github.com/Security-Onion-Solutions/securityonion/issues/3308 to track it.
